src/tests/Dictionary/DictionaryContainer.test.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { mount, shallow } from 'enzyme';
+import { DictionaryOverview, mapStateToProps }from '../../components/dashboard/components/dictionary/DictionaryContainer';
+import dictionary from '../__mocks__/dictionaries';
+import { Router,MemoryRouter,BrowserRouter } from 'react-router-dom';
```

Comment:
what is the point of importing `Router` and `BrowserRouter` if you're not going to use them?